src/staticanalysis/bot/static_analysis_bot/revisions.py

```diff
@@ -161,6 +151,8 @@ def apply(self, repo):
                 )
             except hglib.error.CommandError as e:
                 logger.warning('Failed to update to base revision', revision=self.hg_base, error=e)
+        else:
+            logger.info('No base revision found.')
```

Comment:
I'd log the error when you get baseRevision instead of here

Reply:
Or assert, as discussed in https://github.com/mozilla/release-services/pull/1438/commits/b1057c631ee187e0ffdb40b1ac32bd57e00b5de3#r213327627

Reply:
baseRevision is now required, an execption will be raised from lib_common